.version

```diff
@@ -1 +1 @@
-5.4.1
+5.4.2
```

Comment:
Lmk if you think this should be a minor instead of a patch... There are no big features and nothing breaks so I thought to make it a patch.

Reply:
are `transactionIdentifier` and `productIdentifier` properties that first appeared in this release? If so, that's a minor version, even if they're just a renamed version of the deprecated properties

Reply:
if it's just the deprecation, though... I'm not sure what the semver rules are, we should check

Reply:
yeah, semver says minor even if just deprecating https://semver.org/#spec-item-7